lua/core/project.lua

```diff
@@ -0,0 +1,49 @@
+local M = {}
+function M.config()
+  lvim.builtin.project = {
```

Comment:
Instead of duplicating your default config here, can you instead extend `lvim.builtin.project` ?
Otherwise, if you add entries to your defaults, this is not updated.
This should probably be done for all other plugins ... (not in the scope of this PR)

Reply:
I can't do this unless I am in M.setup() this way I can import the defaults:
```lua
lvim.builtin.project = require("project_nvim.config").defaults
```
But then it would break the autocompletion.

Reply:
Arf, I hate it that we duplicate this, but again it's not specific to this so let's postpone.

Reply:
okay :+1: